test/Git/Index/ParserSpec.hs

```diff
@@ -57,4 +76,17 @@ v2ConflictIndex = "DIRC\NUL\NUL\NUL\STX\NUL\NUL\NUL\STX\
 
 shaV2Cona, shaV2Conb :: Sha1
 shaV2Cona = Sha1.unsafeSha1 "XR\244F9\245-\182}0\173\145C\184j\251\DC4=A_"
-shaV2Conb = Sha1.unsafeSha1 "\ESC-\EOT\135\178\130\210\243\n\160\DLE\EOT6\144\232R\STXl\NAKD"
\ No newline at end of file
+shaV2Conb = Sha1.unsafeSha1 "\ESC-\EOT\135\178\130\210\243\n\160\DLE\EOT6\144\232R\STXl\NAKD"
+
+v2NormalIndex :: BS.ByteString
+v2NormalIndex = "DIRC\NUL\NUL\NUL\STX\NUL\NUL\NUL\SOHZ\
+    \\212\163\235\NUL\174r\148Z\212\163\235\NUL\174r\148\NUL\NUL\b\EOT\
```

Comment:
I put 16 bytes per line for this blob of `stat` information in the example above, but it doesn't really matter...